packages/forklift-console-plugin/src/plans/create/constants.ts

```diff
@@ -0,0 +1,42 @@
+import { t } from 'src/utils/i18n';
```

Comment:
could you please add to tsconfig and webpack the alias of @utils to point to src/utils?